src/richie/apps/courses/models/course.py

```diff
@@ -133,6 +133,11 @@ class Course(BasePageExtension):
         blank=True,
         null=True,
     )
+    is_listed = models.BooleanField(
+        default=True,
+        verbose_name=_("is listed"),
```

Comment:
Just a nitpick but there is no related translation.

Reply:
True! The string will only be available in Crowdin after I merge this PR :man_shrugging: